docker/dev/scripts/start_web.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+set -o pipefail
+
+##############################################################################
+# This script is run by the development Docker image to start the web server
+# for development purposes.
+##############################################################################
+
+# We need webpack to run in the background.
+source venv/bin/activate && yarn webpack &
+# We need to initialize the database.
+source venv/bin/activate && ZEN_ENV=$ZEN_ENV yarn init-db $ZEN_ENV
```

Comment:
Doesn't technically need it - yes - I'm just confused around how how dev's generally use it?

`init-db $ZEN_ENV` - the `$ZEN_ENV` feeds into deployments in `init-dp.py` otherwise it does it for all deployments, not just the specified one.

I'll go with your guidance here - you want me to remove it or keep it?

Reply:
I'd omit init-db here since it doesn't need to run every time the server starts.

It's handy for develop purposes such as if you create a new database migration, then the web server won't start unless you upgrade the database. But it's nice to have to officially upgrade the database if you're working on something incrementally and not have it running automatically. If someone is actively developing against Harmony to the degree they're creating db migrations, then I'd expect them to be comfortable enough with these commands to run `ZEN_ENV=<specify environment> docker compose run web /bin/bash -c "source venv/bin/activate && ./scripts/db/postgres/dev/init_db.py` or `ZEN_ENV=<specify environment> docker compose run web /bin/bash -c "source venv/bin/activate && yarn init-db`.